src/components/TransactionsList.js

```diff
@@ -1,7 +1,23 @@
 import React from 'react'
 
-const TransactionsList = () => {
-
+const TransactionsList = (props) => {
+  
+  let adList = null
```

Comment:
this is not super readable. Could be improved by doing the filtering higher up and then passing down a shorter list.